sdk/azidentity/azidentity_test.go

```diff
@@ -23,6 +23,13 @@ func Test_AuthorityHost_Parse(t *testing.T) {
 
 func Test_SetEnvAuthorityHost(t *testing.T) {
 	err := os.Setenv("AZURE_AUTHORITY_HOST", envHostString)
+	defer func() {
+		// Unset that host environment vairable to avoid other tests failed.
+		err = os.Unsetenv("AZURE_AUTHORITY_HOST")
+		if err != nil {
+			t.Fatalf("Unexpected error when unset environment vairable: %v", err)
```

Comment:
Same nit here, I clearly had trouble with that word the day I wrote this 😂.

Reply:
Fixed (I found a few more cases)